knockout-pre-rendered.js

```diff
@@ -424,8 +512,18 @@
         if (isPlainObject(value) && typeof value['convert'] === 'function') {
             attributeValue = value['convert'](attributeValue);
         }
-      
-        fieldValue[attribute](attributeValue)
+
+        // First try to write to the value as an observable.
+        if (ko.isObservable(fieldValue[attribute])) {
+          fieldValue[attribute](attributeValue)
+        }
+        else {
+          // Otherwise, look for a suitable attribute property writer.
+          var writers = allBindings.get('_ko_prerender_attrPropertyWriters');
```

Comment:
This also looks like a private field. See my previous question.

Reply:
Custom name internal to **knockout-pre-rendered**.